Doc/tools/touch-clean-files.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+"""
+Touch files that must pass Sphinx nit-picky mode
+so they are rebuilt and we can catch regressions.
+"""
+
+from pathlib import Path
+
+CLEAN = [
+    "Doc/library/gc.rst",
+    "Doc/library/sqlite3.rst",
+    "Doc/whatsnew/3.12.rst",
+]
```

Comment:
I know I mentioned this before, but it would be really nice to have this as a static JSON (etc.) data file, or even a simple text file with a blank line between entries (to reduce merge conflicts), rather than part of the dynamic executable code here...

Reply:
What's the benefit of moving it to a static data file? Seems like extra complexity for a simple script, and we're not using the data elsewhere. We could have blank lines in here to avoid conflicts.

Reply:
IMO, but:

* Its cleaner to separate the code and data (especially once the latter dominates)
* Is more concise to write as the length of this list starts to dominate the code in the file (especially if in a simple .gitginore style text format—no indent, no quotes, no comma, etc)
* Is much more more readable/consumable by any other tooling we or others may introduce down the road
* Avoids more complex history and merge conflicts if we ever move it later

None of these are huge reasons on their own, but it seems worth it overall compared to the relatively extra complexity of reading in a gitignore-style simple text file:

```python
CLEAN = Path("clean.txt").read_text(encoding="UTF-8").split("\n")
```

Reply:
Yup, looks great aside from one followup comment :+1: Thanks!